bchrpc/bchrpc.proto

```diff
@@ -431,8 +431,10 @@ message GetUnspentOutputResponse {
     // The index number of the block containing the transaction creating the output.
     int32 block_height = 5;
 
-    SlpToken slp_token = 6;
-    SlpTokenMetadata token_metadata = 7;
+    CashToken cash_token = 6;
```

Comment:
Hmm, maybe this should be added as the last item? Older clients might misinterpret this data since the ordering is changing.